packages/web-twig/tests/components-fixtures/accordionDefault.twig

```diff
@@ -0,0 +1,32 @@
+<Accordion id="AccordionExample">
+    <AccordionItem id="AccordionItemExample0">
+        <AccordionHeader
+            id="AccordionItemExample0Header"
+            for="#AccordionItemExample0Content"
+            slot="<Link href='#'>Link</Link><Pill color='selected'>3</Pill>"
+        >
+            Accordion Header
+        </AccordionHeader>
+        <AccordionContent id="AccordionItemExample0Content" labelId="AccordionItemExample0Header" parent="AccordionExample">
```

Comment:
please fix the labelId here as well and redo the snapshots

Reply:
fixed